src/main/java/seedu/address/logic/Logic.java

```diff
@@ -12,9 +12,10 @@
 import seedu.address.model.entry.Entry;
 
 /**
- * API of the Logic component
+ * API of the Logic component.
```

Comment:
um... you should only use periods when it's a sentence. things like that don't need it but it looks like you added them in for the entire pr anyway so 💀